src/module-3/online-store/styles/base/_global.scss

```
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;900&display=swap');
+
+//map-get($,);
+$colors: (
+  bg-purple: #6F64F8,
+  text-purple: #7E72F2,
+  light-white: #F9F9F9,
+  white: #FFFFFF,
+  light-gray: #EDEDED,
+  border-gray: #979797,
+  divider-gray: #D6D6D6,
+);
+
+$sizes: (
+  xs: 320px,
+  sm: 576px,
+  md: 768px,
+  lg: 992px,
+  xl: 1200px,
+  xxl: 1850px);
+
+//@include mixin();
+@mixin media($media) {
+  @each $size, $resolution in $sizes {
+    @if $media == $size {
+      @media (#{max}-width: $resolution) {
+        @content;
+      }
+    }
+  }
+}
+
+@mixin def-font($font-size, $font-weight, $color:black) {
```

```suggestion
@mixin def-font($font-size, $font-weight, $color: inherit) {
```  
this is a more expected variant